src/main/drivers/rx/rx_sx1280.h

```diff
@@ -175,6 +180,75 @@ typedef enum {
     SX1280_LORA_CRC_OFF = 0x00, // CRC not used
 } sx1280LoraCrcModes_e;
 
+/*!
+ * \brief Represents the bandwidth values for FLRC packet type
+ */
+typedef enum {
+    SX1280_FLRC_BR_1_300_BW_1_2 = 0x45,
+    SX1280_FLRC_BR_1_000_BW_1_2 = 0x69,
+    SX1280_FLRC_BR_0_650_BW_0_6 = 0x86,
+    SX1280_FLRC_BR_0_520_BW_0_6 = 0xAA,
+    SX1280_FLRC_BR_0_325_BW_0_3 = 0xC7,
+    SX1280_FLRC_BR_0_260_BW_0_3 = 0xEB,
+} SX1280_RadioFlrcBandwidths_t;
+
+/*!
+ * \brief Represents the coding rate values for FLRC packet type
+ */
+typedef enum {
+    SX1280_FLRC_CR_1_2 = 0x00,
+    SX1280_FLRC_CR_3_4 = 0x02,
+    SX1280_FLRC_CR_1_0 = 0x04,
+} SX1280_RadioFlrcCodingRates_t;
+
+/*!
+ * \brief Represents the Gaussian filter value in FLRC packet types
+ */
+typedef enum {
+    SX1280_FLRC_BT_DIS  = 0x00,
+    SX1280_FLRC_BT_1    = 0x10,
+    SX1280_FLRC_BT_0_5  = 0x20,
+} SX1280_RadioFlrcGaussianFilter_t;
+
+typedef enum {
+    SX1280_FLRC_SYNC_NOSYNC        = 0x00,
+    SX1280_FLRC_SYNC_WORD_LEN_P32S = 0x04,
+} SX1280_RadioFlrcSyncWordLen_t;
+
+typedef enum {
```

Comment:
This is just the table from the datasheet copied over so I left it. I think the idea is for some registers some combinations are not valid, so using the enum would be preferable to prevent creating an invalid combination?

Reply:
At least it is only 3 bits, so 2^n is small ..